bin/test_flakiness

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+
+# Determine how flaky our UI tests by measuring recent failure rate
+# using data from the Saucelabs API. Prints out every UI test from
+# least flaky to most flaky.
+
+require_relative '../deployment'
+require 'rest-client'
+
+username = ENV['SAUCE_USERNAME'] || CDO.saucelabs_username
+key = ENV['SAUCE_ACCESS_KEY'] || CDO.saucelabs_authkey
+
+PER_REQUEST = 1500 # maximum returned per API call (undocumented)
+NUM_REQUESTS = 50 # rate limit: 15 request/s with 300 request burst https://wiki.saucelabs.com/display/DOCS/Rate+Limits+for+the+Sauce+Labs+REST+API
```

Comment:
How are we making sure that we're not exceeding the request rate limit? Or are we assuming that the call is slow enough that making synchronous calls is limiting enough.